tchannel/zipkin/tracers.py

```diff
@@ -148,10 +153,18 @@ def submit_callback(f):
                     exc_info=f.exc_info()
                 )
 
+        span_host = self.parse_host_port()
```

Comment:
what's the reason not to do it in `__init__`?  We already have tchannel by then.